sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>Is it possible to use sagemaker.image_uris.retrieve?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Not possible to use retrieve() function